blueflood-http/src/integration-test/java/com/rackspacecloud/blueflood/http/HttpEnumIntegrationTest.java

```diff
@@ -16,29 +16,43 @@
 
 package com.rackspacecloud.blueflood.http;
 
+import com.rackspacecloud.blueflood.inputs.formats.JSONMetricsContainerTest;
+import com.rackspacecloud.blueflood.inputs.handlers.wrappers.AggregatedPayload;
+import com.rackspacecloud.blueflood.service.Configuration;
+import com.rackspacecloud.blueflood.service.CoreConfig;
 import com.rackspacecloud.blueflood.service.EnumValidator;
 import com.rackspacecloud.blueflood.types.Locator;
 import org.apache.http.HttpResponse;
 import org.apache.http.util.EntityUtils;
-import org.junit.Assert;
 import org.junit.Test;
 
+import java.io.IOException;
+import java.net.URISyntaxException;
 import java.util.HashSet;
 import java.util.Set;
+import java.util.regex.Pattern;
+
+import static org.junit.Assert.*;
+
+import static org.junit.Assert.assertTrue;
 
 public class HttpEnumIntegrationTest extends HttpIntegrationTestBase {
 
     @Test
     public void testMetricIngestionWithEnum() throws Exception {
+
         // ingest and rollup metrics with enum values and verify CF points and elastic search indexes
         final String tenant_id = "333333";
-        final String metric_name = "enum_metric_test";
+
+        String prefix = getPrefix();
+        final String metric_name = prefix + "enum_metric_test";
```

Comment:
Just curious, why did you put the random number as prefix, as supposed to suffix? Doesn't that make it harder for you to search things in Cassandra or ES when you want to verify if the metrics are in those systems?